Frameworks/Plugins/ZappCrashlogsMsAppCenter/Podfile

```diff
@@ -12,7 +12,7 @@ end
 target 'FrameworksApp' do
   supports_swift_versions '5.1'
   use_frameworks!
-  pod 'ZappCrashlogsMsAppCenter', :path => '../../../ZappCrashlogsMsAppCenter.podspec', :testspecs => ['UnitTests']
+  pod 'ZappCrashlogsMsAppCenter', :path => 'Files/ZappCrashlogsMsAppCenter.podspec', :testspecs => ['UnitTests']
   pod 'ZappCore', :path => '../../../ZappCore.podspec'
```

Comment:
will this path always resolve properly when we're building the app ? from what I understand, we are assuming here that when we are building the ZappCrashlogsMsAppCenter pod, we expect ZappCore's podspec file to be 3 directories up. Will that always be the case ?